invenio_app_ils/config.py

```diff
@@ -95,6 +95,9 @@
     EITEM_PID_FETCHER,
     EITEM_PID_MINTER,
     EITEM_PID_TYPE,
+    FRONT_SITE_SEARCH_PID_FETCHER,
```

Comment:
What about `RECORDS` or `ALL_RECORDS` or similar? I am not sure we want to have it "tight" to the front site.
It is more for patrons in this case... `PATRONS_DOCS`?